src/components/Beer.js

```diff
@@ -0,0 +1,57 @@
+import './beer.css'
+import { useParams  } from "react-router-dom"
+import { useEffect } from 'react'
+import { useState } from 'react/cjs/react.development'
```

Comment:
useState & useEffect are both imported from 'react', so you can combine the imports

```
import { useEffect, useState } from 'react'
```